sdcm/cluster.py

```diff
@@ -4147,6 +4147,7 @@ def check_cluster_health(self):
                 return
 
             self.check_nodes_running_nemesis_count()
+            self.test_config.tester_obj().validate_partitions()
```

Comment:
so what is expected result in case `delete_rows` nemesis ran?

Reply:
It's expected for `delete_rows` to delete rows other than the ones of `partition_range_with_data_validation`, so the validation should succeed without any impact.